UKHO.FmEssFssMock.API/UKHO.FmEssFssMock.API/Services/ExchangeSetService.cs

```diff
@@ -90,10 +141,11 @@ private ExchangeSetServiceResponse GetEssResponse(string responseId)
         private CreateBatchRequest CreateBatchRequestModel(bool isPostProductIdentifiersRequest)
         {
             PosTestCase currentTestCase = _mockService.GetCurrentPOSTestCase(_homeDirectoryPath);
+            string batchType;
 
-            string batchType = currentTestCase != PosTestCase.ValidProductIdentifiers
-                ? currentTestCase.ToString()
-                : isPostProductIdentifiersRequest ? Batch.EssFullAvcsZipBatch.ToString() : Batch.EssUpdateZipBatch.ToString();
+            batchType = currentTestCase != PosTestCase.ValidProductIdentifiers
```

Comment:
Remove line 144 and declare directly here

Split out conditional expression for if/else statements to improve readability

Reply:
Resolved